.travis.yml

```diff
@@ -6,8 +6,10 @@ scala:
   - "2.12.8"
   - "2.11.12"
 
-jdk:
-  - oraclejdk8
+before_install:
+  # using jabba for custom jdk management
+  - curl -sL https://raw.githubusercontent.com/shyiko/jabba/0.11.1/install.sh | bash && . ~/.jabba/jabba.sh
+  - jabba install adopt@1.8.192-12
```

Comment:
Is that correct? you have the curl pulling `0.11.1` and installing `adopt@1.8.192-12`

Reply:
0.11.1 is the version of jabba

here is an example that is using both jdk 8 and 11: https://github.com/akka/akka-management/pull/427/files#diff-354f30a63fb0907d4ad57269548329e3R11